scripts/build_wasm.sh

```diff
@@ -7,9 +7,6 @@ fi
 
 parachain_name=$1
 
-# TODO: subwasm always missing on epyc even though it's installed
-cargo install --locked --git https://github.com/chevdor/subwasm --tag v0.16.1
```

Comment:
Why removing this?

Reply:
It takes time to build it with cargo and we already have it on our self hosted github runners.